.gitignore

```diff
@@ -8,3 +8,9 @@ vendor/
 
 # Mac OS
 .DS_Store
+
+# Jekyll
```

Comment:
This are all added already

Reply:
Thanks! I didn't see that. Solved in [90cf71f](https://github.com/inbo/craywatch/pull/177/commits/90cf71f1bfa834e7c437876c118088cfa4d985a2). I added comment `# Jeykyll` on top so that these block is more visible.